concordia/templates/django_registration/registration_form.html

```diff
@@ -8,46 +8,30 @@
 <div class="container-fluid">
     <div id="registration-form-container" class="row flex-md-row justify-content-center">
         <div class="col-md-6 p-3">
+            <h2 class="mt-0 text-center">Make Your Mark</h2>
+            <h3 class="mb-default text-center">Registration</h3>
             <p>
-                Create an account to be able to track your work, add tags
-                and review transcriptions. Please provide a username, your
-                email address and a strong password.
-            </p>
-            <hr />
-            <p>
-                Why do we need your email? Your email address gives us the
-                ability to support you. Community managers are here to help
-                with account administration like changes to your profile,
-                troubleshooting any issues with contributing to a
-                transcription, or to answer general questions. You have the
-                additional option to receive email updates by opting to
-                subscribe. We will never share your information with other
-                institutions or individuals.
+                Register for an account to track your work, add tags and review transcriptions. To sign up, please provide a username, email address and a strong password.
+                Once you click register, we will send you an email to confirm your address.
             </p>
-            <hr />
+                Learn more about why we ask you to register in the <a href="{% url 'about' %}">Make Your Mark FAQ</a>
             <p>
-                Once you click register, we will send you a link to confirm
-                your email address.
+                <hr />
```

Comment:
`<hr>` cannot be nested in a `<p>` according to the HTML standard (I just tested this on validator.w3.org)